cmd/hypper/testdata/output/install-with-shared-deps-out-of-range.txt

```diff
@@ -0,0 +1,3 @@
+🛳  Installing shared dependencies for chart "empty":
+ERROR: ❌  Satisfiable version for chart "testdata/testcharts/shared-dep" not found, aborting
+Error: Satisfiable chart version not found; 0.1.0 is less than 0.3.0
```

Comment:
I don't like this output at all;
I thought that line 2 should just be red (it is) but not contain the `ERROR` in the beginning.
Also, line 3 is a legit error that has been floated upwards, I suppose we are forgetting to print it through our logger. I suppose we can fix it in a following PR.

Reply:
You are right, we should add a cart for that (e.g. expand the style guide to include colouring of output).

Reply:
opened https://github.com/rancher-sandbox/hypper/issues/155 !